modules/client-configuration/pages/client-proxy-script.adoc

```diff
@@ -0,0 +1,69 @@
+[[script-client-proxy]]
+= Register Clients  to a Proxy with a Bootstrap Script
+
+
+You can register clients through the {productname} Proxy with a bootstrap script.
+This is done almost the same way as registering clients directly with the {productname} Server.
+
+You create the bootstrap script on the {productname} Server with a command line tool.
+The bootstrap script then deploys all necessary information to the clients.
+The bootstrap script requires some parameters such as the FQDN of the proxy, activation keys, or GPG keys.
+These parameters depend on your specific setup, but the FQDN of the proxy is mandatory.
```

Comment:
This is little confusing. It's not a bootstrap script requiring that parameters, but the script generating bootstrap script.